.drone.yml

```diff
@@ -9,7 +9,15 @@ pipeline:
       event:
         - push
         - tag
-
+  scan:
+    image: quay.io/ukhomeofficedigital/anchore-submission:latest
+    dockerfile: Dockerfile
+    image_name: quay.io/ukhomeofficedigital/etcd-backup:${DRONE_COMMIT_SHA}
+    tolarates: medium
```

Comment:
`tolarates` => `tolerates` (the typo is still valid in anchore-submission:latest but better to correct it)